packages/plans-grid-next/src/components/features-grid/index.tsx

```diff
@@ -260,7 +260,7 @@ const TabletView = ( {
 	const gridPlansWithoutSpotlight = ! gridPlanForSpotlight
 		? renderedGridPlans
 		: renderedGridPlans.filter( ( { planSlug } ) => gridPlanForSpotlight.planSlug !== planSlug );
-	const numberOfPlansToShowOnTop = 4 === gridPlansWithoutSpotlight.length ? 2 : 3;
+	const numberOfPlansToShowOnTop = 4 === gridPlansWithoutSpotlight.length ? 4 : 3;
```

Comment:
If there are only 4 plans, they can be shown on a single row without issues.

https://github.com/user-attachments/assets/9a93807c-d537-4fe3-9bbe-11e88db243bc